homeassistant/components/omnilogic/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This file is outdated

Reply:
I think I've addressed....

Reply:
You're not supposed to edit by hand. Use `hassfest` to generate this for you

Reply:
I will generate the file using the script, and then add the correct English translations in the next commit.

Reply:
There's no need to change anything in the generated file. The replacement of the % values will happen automatically.

Reply:
Where does it get the translations? When I use just the generated en.json file in my test environment it _does not_ replace any of the % values. Also there is some guidance required on the username for the Omnilogic login - they use email address for some of their logins and username for others.

Reply:
The % values are replaced as part of an automatic task - you're not going to see it in production.
If you need custom text for the username, you have two options:
1. Not use a % value, but something custom of your own. There's a limited number of % values, and you're encouraged to use them when using common values (username, password, etc...), but you don't have to
2. Use a description in the dialog to instruct the user on what username to use instead of using the field for this.

Reply:
OK so just leave the generated file. Will do.

Reply:
this needs to be re-generated (another option is to simply remove it from git - it will be created automatically next time the localization task runs)

Reply:
This needs to either be re-generated or removed

Reply:
Reminding you that this needs to be either generated again or removed